components/Modal/CreateImageUpload.js

```diff
@@ -0,0 +1,751 @@
+                  })}
+                  fieldId={key}
+                  key={key}
+                >
+                  <TextInput
+                    value={this.state.uploadSettings[key]}
+                    type="text"
+                    id={key}
+                    key={key}
+                    onChange={this.setUploadSettings}
+                  />
+                </FormGroup>
+              ))}
+            </Form>
+          )}
+        </React.Fragment>
+      )
+    };
+
+    const uploadOpenstackStep = {
+      name: "Upload to Openstack",
+      steps: [uploadOpenstackAuth, uploadOpenstackSettings]
+    };
+
+    const reviewStep = {
+      name: "Review",
+      component: (
+        <TextContent>
+          <Title headingLevel="h3" size="2xl">
+            <FormattedMessage defaultMessage="Create and Upload Image" />
+          </Title>
+          <Text>{formatMessage(messages.review)}</Text>
+          {providerSettings[uploadService] !== undefined && (
+            <TextList component={TextListVariants.dl}>
+              <TextListItem component={TextListItemVariants.dt}>
+                <FormattedMessage defaultMessage="Image Name" />
+              </TextListItem>
+              <TextListItem component={TextListItemVariants.dd}>{imageName}</TextListItem>
+              {Object.keys(providerSettings[uploadService].auth).map(key => (
+                <>
+                  <TextListItem component={TextListItemVariants.dt}>
+                    {formatMessage({
+                      id: "review-settings",
+                      defaultMessage: providerSettings[uploadService].auth[key]
+                    })}
+                  </TextListItem>
+                  <TextListItem component={TextListItemVariants.dd}>{this.state.uploadSettings[key]}</TextListItem>
+                </>
+              ))}
+              {Object.keys(providerSettings[uploadService].settings).map(key => (
+                <>
+                  <TextListItem component={TextListItemVariants.dt}>
+                    {formatMessage({
+                      id: "review-settings",
+                      defaultMessage: providerSettings[uploadService].settings[key]
+                    })}
+                  </TextListItem>
+                  <TextListItem component={TextListItemVariants.dd}>{this.state.uploadSettings[key]}</TextListItem>
+                </>
+              ))}
+            </TextList>
+          )}
+        </TextContent>
+      )
+    };
+
+    const steps = [
+      imageStep,
+      ...(showUploadAzureStep ? [uploadAzureStep] : []),
+      ...(showUploadOpenstackStep ? [uploadOpenstackStep] : []),
+      ...(showUploadVSphereStep ? [uploadVSphereStep] : []),
+      ...(showReviewStep ? [reviewStep] : [])
+    ];
+
+    const createImageUploadFooter = (
+      <WizardFooter>
+        <WizardContextConsumer>
+          {({ activeStep, onNext, onBack, onClose }) => {
+            return (
+              <>
```

Comment:
Please stick to either this syntax or <React.Fragment>, don't use them interchangeably

Reply:
Fixed